extensions/azurePublish/src/node/index.ts

```diff
@@ -673,6 +703,52 @@ export default async (composer: IExtensionRegistration): Promise<void> => {
       return recommendedResources;
     };
 
+    private isProfileProvisioned = (profile: PublishConfig) => {
```

Comment:
to make sure this function returns boolean, you can add a boolean return type and fix linter complains 😄